db/schema.rb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20160718141156) do
+
+  create_table "airports", force: :cascade do |t|
+    t.string   "name"
+    t.string   "country"
+    t.string   "state"
+    t.string   "airport_code"
+    t.string   "type"
+    t.integer  "route_id"
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+  end
+
+  add_index "airports", ["route_id"], name: "index_airports_on_route_id"
+
+  create_table "bookings", force: :cascade do |t|
+    t.string   "reference_number", null: false
+    t.integer  "flight_id"
+    t.integer  "user_id"
```

Comment:
Remove unregistered_user model and dissociated it from booking model using migrations